requirements.txt

```diff
@@ -1,3 +1,4 @@
+black~=20.8b1
```

Comment:
Would it be worthwile to split the lint dependencies separately? That at least should be our practice for csh-ldap to limit dependency conflicts.

Reply:
I don't really see that being useful, if anything it would be annoying especially in ide's and editors like vscode

Reply:
Worth keeping in mind for csh_ldap.